composer.json

```diff
@@ -17,25 +17,25 @@
     "require": {
         "php": "^8.2",
         "ext-json": "*",
-        "illuminate/console": "^11.15.0",
-        "illuminate/container": "^11.15.0",
-        "illuminate/contracts": "^11.15.0",
-        "illuminate/database": "^11.15.0",
-        "illuminate/http": "^11.15.0",
-        "illuminate/pipeline": "^11.15.0",
-        "illuminate/support": "^11.15.0",
+        "illuminate/console": "^11.41.3 || ^12.0",
+        "illuminate/container": "^11.41.3 || ^12.0",
+        "illuminate/contracts": "^11.41.3 || ^12.0",
+        "illuminate/database": "^11.41.3 || ^12.0",
+        "illuminate/http": "^11.41.3 || ^12.0",
+        "illuminate/pipeline": "^11.41.3 || ^12.0",
+        "illuminate/support": "^11.41.3 || ^12.0",
```

Comment:
Why are we also increasing the minimum 11.x version?

Reply:
The minimum 11.x version was updated to fix compatibility issues on higher 11.x versions.

In my opinion, it is much cleaner and maintainable to fix the issue and raise the minimum version than to create two **nearly identical** files and have an if condition in the GeneralTypeTest like you did here: https://github.com/larastan/larastan/commit/cef7cfdfe6cf0905823e932e24046175dc6d05bd

Reply:
I'll update since you insist, however, this is not very useful given that a `composer update` is going to update people to the latest 11.x---there's no reason for anyone to not be on the latest minor version.